module/behaviour/skills/KickScript/src/KickScript.cpp

```diff
@@ -61,52 +62,32 @@ namespace module::behaviour::skills {
         });
 
         on<Trigger<KickScriptCommand>>().then([this](const KickScriptCommand& kickCommand) {
-            auto direction = kickCommand.direction;
-            LimbID leg     = kickCommand.leg;
-
-            int quadrant = getDirectionalQuadrant(direction[0], direction[1]);
-
-            // check if the command was valid
-            bool valid = true;
-            if (leg == LimbID::RIGHT_LEG) {
-                if (quadrant == 2 || quadrant == 3) {
-                    NUClear::log<NUClear::WARN>("Right leg cannot kick towards: ", direction);
-                    valid = false;
-                }
-            }
-            else if (leg == LimbID::LEFT_LEG) {
-                if (quadrant == 2 || quadrant == 1) {
-                    NUClear::log<NUClear::WARN>("Left leg cannot kick towards: ", direction);
-                    valid = false;
-                }
-            }
-            else {
-                NUClear::log<NUClear::WARN>("Cannot kick with limb: ", leg);
-                updatePriority(0);
-                valid = false;
-            }
-
-            if (valid) {
-                this->kickCommand = kickCommand;
-                updatePriority(KICK_PRIORITY);
-            }
+            this->kickCommand = kickCommand;
+            updatePriority(KICK_PRIORITY);
         });
 
         on<Trigger<ExecuteKick>>().then([this] {
             // auto direction = kickCommand.direction;
             LimbID leg = kickCommand.leg;
 
-            if (leg == LimbID::RIGHT_LEG) {
-                emit(std::make_unique<ExecuteScriptByName>(
-                    id,
-                    std::vector<std::string>({"Stand.yaml", "KickRight.yaml", "Stand.yaml"})));
+            // Execute the penalty kick if the type is PENALTY
+            if (kickCommand.type == KickCommandType::PENALTY) {
+                emit(std::make_unique<ExecuteScriptByName>(id, std::vector<std::string>({"KickPenalty.yaml"})));
```

Comment:
How do we know what foot we kick with? This shouldn't be webots specific.

Reply:
This implementation is only temporary to get it working in webots. Proper implementation of penalty shootout behaviour is out of scope of this PR.

Reply:
I can comment this in, but adding the ability to change what foot we kick with won't be useful atm.

Reply:
Comments are good.